packages/nuxt/src/app/composables/router.ts

```diff
@@ -92,6 +111,26 @@ export const navigateTo = (to: RouteLocationRaw | undefined | null, options?: Na
   }
 
   const toPath = typeof to === 'string' ? to : ((to as RouteLocationPathRaw).path || '/')
+
+  // Early open handler
+  if (options?.open) {
+    if (process.client) {
+      const { target = '_blank', windowFeatures = {} } = options.open
+
+      const features = Object.entries(windowFeatures)
+        .map(([feature, value]) => {
+          const formattedValue =
+              typeof value === 'boolean' ? (value ? 'true' : 'false') : value
+          return `${feature.toLowerCase()}=${formattedValue}`
```

Comment:
I think implicit conversion to string should work to convert boolean -> string.

```suggestion
          return `${feature.toLowerCase()}=${value}`
```

Reply:
Sorry for overwrite your changes, forgot to put this on draft.

Checking this now